corehq/apps/toggle_ui/utils.py

```diff
@@ -1,7 +1,108 @@
+from couchexport.writers import Excel2007ExportWriter
+from django.utils.translation import ugettext as _
+from io import BytesIO
 from corehq.toggles import all_toggles
 
+from corehq.apps.accounting.models import Subscription
+from corehq.apps.es import UserES
+from corehq.util.quickcache import quickcache
+
 
 def find_static_toggle(slug):
     for toggle in all_toggles():
         if toggle.slug == slug:
             return toggle
+
+
+@quickcache(['domain'], timeout=10)
+def get_subscription_info(domain):
+    subscription = Subscription.get_active_subscription_by_domain(domain)
+    if subscription:
+        return subscription.service_type, subscription.plan_version.plan.name
+    else:
+        return None, None
+
+
+@quickcache(['domain'], timeout=10)
+def has_dimagi_user(domain):
+    return UserES().web_users().domain(domain).search_string_query('@dimagi.com').count()
+
+
+def parse_flags_to_file_info(toggles):
+    def get_header(slug):
```

Comment:
This hardly seems like it needs a function, you could just as well do:

```
toggle_headers = [(
            _("Label"),
            _("Slug"),
            _("Tag"),
            _("Documention link"),
            _("Description")
        )]

for ...:
    file_headers.append((toggle.slug, toggle_headers))

```

Reply:
Yeah, this seems like a better approach. Function is maybe a bit overkill...